EIPS/eip-5560.md

```diff
@@ -0,0 +1,122 @@
+---
+eip: 5560
+title: Redeemable NFTs
+description: Links physical objects to NFTs
```

Comment:
[Editorial] It is hard for me to connect the description with the title, could you help elaborate the "redeemable" aspect in the description?

Reply:
Thanks, I suggested "Makes an NFT redeemable for a physical object".